### ISSUE TITLE
Human Evaluation Results

### ISSUE THREAD

**Issue by JundaHe** [gmy2013/LLM_Comment_Generation#1]
Hi! Is it possible for you to release the original result of the human evaluation mentioned in the paper?

**Comment by gmy2013** [comment]
Of course! Since CodeX is depreciated, we are currently testing the results of GPT-3.5-turbo and will release the generated results along with the human evaluation result in the updated version.
